### PR TITLE
add first package and demo

### PR DESCRIPTION
**Pull request by mm25zamanian**

### CODE REVIEW

File: demo/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "composite": true,
+    "rootDir": ".",
+    "outDir": ".",
+    "tsBuildInfoFile": ".tsbuildinfo"
+  },
+  "include": [
+    "**/*.ts"
+  ],
+  "exclude": [
+    "dist",
```
Comment (asmanp2012) [inline demo/tsconfig.json:13]: change `"dist",` to `"*.d.ts",`

File: package/util/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@one/util",
+  "version": "0.0.0",
+  "description": "Typescript/Javascript Utils",
+  "main": "./dist/index.js",
```
Comment (asmanp2012) [inline package/util/package.json:5]: change main to `index.js`

File: package/util/package.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  "author": "@one community",
+  "contributors": [
+    "MohammadMahdi Zamanian <mm25zamanian@gmail.com>"
+  ],
+  "dependencies": {
+    "tslib": "^2.2.0"
```
Comment (asmanp2012) [inline package/util/package.json:11]: change `"tslib": "^2.2.0"` to `"tslib": "^2.3.1"`

File: package/util/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    "tsBuildInfoFile": ".tsbuildinfo"
+  },
+  "include": [
+    "**/*.ts"
+  ],
+  "exclude": [
+    "*.d.ts",
+    "node_modules"
+  ]
+}
```
Comment (asmanp2012) [inline package/util/tsconfig.json:16]: Add break end of the file.

File: demo/package.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@one/demo",
+  "version": "0.0.0",
+  "description": "@one packages demo",
+  "main": "./dist/index.js",
```
Comment (asmanp2012) [inline demo/package.json:5]: change `"main": "./dist/index.js",`  to `"main": "index.js",`

File: demo/src/util/index.html [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<html lang="en">
```
Comment (asmanp2012) [inline demo/src/util/index.html:1]: Move all of the src file and directory to the root demo package directory.

Comment (asmanp2012) [inline demo/src/util/index.html:1]: Change to this
```HTML
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title></title>
</head>

<body>
  <h3>Check the console</h3>
  <script src="./index.js" type="module"></script>
</body>

</html>
```

File: demo/package.json (hunk 5 above)
Comment (asmanp2012) [inline demo/package.json:5]: Add `"type": "module",`

File: package/util/package.json (hunk 2 above)
Comment (asmanp2012) [inline package/util/package.json:5]: Add `"type": "module",`

**Review by asmanp2012 — CHANGES_REQUESTED** [review]
Move all of the src files and directory to the root demo package directory.

File: package.json [hunk 7]
```diff
@@ -3,14 +3,27 @@
...
   "private": true,
+  "scripts": {
+    "b": "yarn build",
+    "w": "yarn watch",
+    "s": "yarn serve",
+    "c": "yarn clean",
+    "build": "npm-run-all build:*",
+    "build:ts": "tsc --build",
+    "prepare": "yarn build",
+    "watch": "npm-run-all --parallel watch:*",
```
Comment (asmanp2012) [inline package.json:19]: Remove all `watch` scripts.

File: demo/tsconfig.json [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+    "node_modules"
+  ]
+}
```
[the remark this answers is not in the document]
Comment (asmanp2012) [reply demo/tsconfig.json:16]: ?

File: demo/src/util/index.html (hunk 6 above)
Comment (asmanp2012) [reply demo/src/util/index.html:1]: ?

File: demo/src/index.html [hunk 9]
```diff
@@ -0,0 +1,13 @@
+<html lang="en">
```
Comment (asmanp2012) [inline demo/src/index.html:1]: Remove `src` and move all content to the root `demo` package directory.

**Review by asmanp2012 — CHANGES_REQUESTED** [review]

**Comment by mm25zamanian** [comment]
Ok ???
but please review eslint config !!!!

File: demo/util/index.ts [hunk 10]
```diff
@@ -0,0 +1,3 @@
+import { persianNumberToEnglishNumber } from "@one/util/src/persian-number";
```
Comment (asmanp2012) [inline demo/util/index.ts:1]: change `@one/util/src/persian-number` to `@one/util/persian-number`

File: package.json [hunk 11]
```diff
@@ -3,31 +3,39 @@
   "version": "0.0.0",
   "description": "One Web Components",
   "repository": "https://github.com/AliMD/One",
-  "author": "Ali Mihandoost <alimihandoost@gmail.com>",
+  "author": "@one community",
   "contributors": [
     "MohammadMahdi Zamanian <mm25zamanian@gmail.com>",
     "Amir Najafi <njfamirm@gmail.com>"
   ],
+  "license": "MIT",
```
Comment (asmanp2012) [inline package.json:11]: Remove `license`

File: package.json [hunk 12]
```diff
@@ -3,31 +3,39 @@
...
+    "b": "yarn build",
+    "s": "yarn serve",
+    "c": "yarn clean",
+    "build": "npm-run-all lint build:*",
+    "build:ts": "tsc --build",
     "lint": "npm-run-all lint:*",
-    "lint:ts": "eslint package/**/*.ts **/*.ts"
+    "lint:ts": "eslint package/**/*.ts **/*.ts",
+    "prepare": "yarn build",
+    "serve": "node ./dist/src/server.js",
```
Comment (asmanp2012) [inline package.json:22]: remove `serve`

**Review by asmanp2012 — CHANGES_REQUESTED** [review]
remove extra scripts and fix import addres

**Comment by mm25zamanian** [comment]
@asmanp2012 Please Merge pull !

**Review by asmanp2012 — APPROVED** [review]
All config is ok and doesn't have any problem.

**Pull request merged**